### DISCUSSION TITLE
Revive the project?

### DISCUSSION THREAD

**Discussion by JaeGif** [club-40/odin-pantry#39] [General]
Anyone willing to revive this project? If there's even just 2 or 3 of us, we can probably manage it in a reasonable time frame.

I'm personally in a place where I can contribute a decent amount of time to this project ~15-20hrs a week. If there's a couple more people willing maybe we can start a discord channel and discuss it?

**Comment by Philip-Clark** [comment]
@JaeGif Sounds great. I'm not able to commit time to it, but I think it's a great project and think a lot can be learned from working on this. I learned a lot just through the little bit I did. 

You might also try bumping the original thread on TOP's discord [here](https://discord.com/channels/505093832157691914/1091794976045813770) to generate more interest. 

Feel free to run with this!

( Note: If you do decide to head up something, shoot me an email, Discord DM, or tag me so I can promote your privileges and role here. )

**Comment by RyanMcEntire** [comment]
I assumed this project was finished. Back when it was going on I didn't know React so I felt like I couldn't contribute anything. I've learned a decent amount since then, and i have experience with typescript so i'm interested in seeing what I can do to help if this is still in the works.

**Comment by Philip-Clark** [comment]
Ryan has volunteered to head the project moving forward. I've added them to the maintainer's team and given them the owner role for full admin permission.

@RyanMcEntire I appreciate your initiative! I'm looking forward to what you all achieve! 💪🚀

**Comment by RyanMcEntire** [comment]
Thanks for all the work you've done @Philip-Clark ! You've organized the groundwork. We'll do ya proud.
